test.md

```diff
@@ -16,17 +16,18 @@ Assertions
 These assertions will check the supplied property, and then clear that state from the configuration.
 In this way, tests can be written as a serious of setup, execute, assert cycles which leaves the configuration empty on success.
 
-We'll make `Assertion` a subsort of `Instr`, so that we can easily consume `Instr` with `trap`s, but not consume `Assertion`s.
+We'll make `Assertion` a subsort of `Auxil`, so that we can easily consume `Instr` with `trap`s, but not consume `Assertion`s.
```

Comment:
`Instr` should be changed to `Auxil`

Reply:
I think we are consuming `Instr` not `Auxil`?

Reply:
Sorry, I meant `Instr` and `Auxil` should be changed to `Stmt` or whatever our top level is, right? So a trap consumes everything that is top level but isn't an assertion. Or at least it should consume both `Instr` and `Defn`.

Reply:
Fixed.